### DISCUSSION TITLE
Where is the SQLite database stored?

### DISCUSSION THREAD

**Discussion by leowankerddd** [vzhd1701/evernote-backup#55] [Q&A] [answered]
Hi all,

After I'm done using this software, where can I find the database? It will be taking up lots of disk space I'd like to recover.

Thanks.

**Comment by vzhd1701** [comment, marked as answer]
By default it is stored in `en_backup.db` file in the working directory where you executed `evernote-backup` from.

**Reply by leowankerddd** [reply]
Thank you very much for your prompt reply!
